lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -95,6 +101,11 @@ def main():
         'n_pixels',
         'wl',
         'r',
+        'log_intensity',
+        'x',
+        'y',
+        'disp_dx',
+        'disp_dy',
```

Comment:
I think the test is not passing because these parameters are not being defined later on, `dl1_container.disp_dx = ...`, etc